lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -33,19 +33,19 @@ class AxeAudit extends Audit {
     // Indicate if a test is not applicable.
     // This means aXe did not find any nodes which matched these checks.
     // Note in Lighthouse we use the phrasing "Not Applicable" (aXe uses "inapplicable", which sounds weird).
-    const notApplicables = artifacts.Accessibility.notApplicable || [];
+    const notApplicables = artifacts.Accessibility.notApplicable;
     const isNotApplicable = notApplicables.find(result => result.id === this.meta.id);
     if (isNotApplicable) {
       return {
-        score: 1,
+        score: null,
```

Comment:
threw this one in for https://github.com/GoogleChrome/lighthouse/pull/12075#discussion_r573986039 and because technically I did this, when I was updating old `rawValue: true`s to `score: 1`s all over the place :)